src/user-agent.ts

```diff
@@ -4,4 +4,15 @@
  * user agent of sdk
  * @hidden
  */
-export const USER_AGENT = "sdk-js-2.0.0-alpha.18";
+import { isSSR } from "./index";
+
+const scriptTag = (document.currentScript as HTMLScriptElement | null)?.src;
```

Comment:
Is `document` defined in an SSR world?

Reply:
It wouldn't be, no. It probably needs a test as you mentioned. I'm just not sure if it's possible in Jest? I'm sure someone's worked it out. We could do with some SSR tests in the React SDK too.

Reply:
You can mock the `isSSR` function at least cos you can mock any modules. But yeah there's probably a way to test cases where document may or may not be defined.